.github/workflows/pvcviewer_controller_unit_test.yaml

```
@@ -19,7 +19,7 @@ jobs:
     - name: Install Go
       uses: actions/setup-go@v5
       with:
-        go-version: "1.22.2"
+        go-version: "1.24"
```

```suggestion
        go-version-file: 'components/pvcviewer-controller/go.mod'
```

Given we already need to adjust this code block - I think we would be better served to change it more _substantially_ to avoid having to hardcode (and couple) this `go-version` _input_ with the related `go.mod` file

You can see this pattern employed in `notebooks-v2` code - and I think we should have `notebooks-v1` follow suit!  Will make any future updates easier - as we only need to modify `go.mod` and the GH workflow will then automatically pick up the right version.
- https://github.com/kubeflow/notebooks/blob/notebooks-v2/.github/workflows/ws-backend-test.yml#L31
    - note: for now lets **NOT** include the `cache` related key here - even though it is present in `notebooks-v2`